libs/SmartStore/Configuration/SmartStore-Common.xcconfig

```diff
@@ -11,8 +11,8 @@ GCC_PREFIX_HEADER = $(PROJECT_NAME)/$(PROJECT_NAME)-Prefix.pch
 
 PRODUCT_NAME = SmartStore
 
-OTHER_CFLAGS = -DSQLITE_HAS_CODEC -DFMDatabase=SF_FMDatabase -DFMStatement=SF_FMStatement -DFMDatabasePool=SF_FMDatabasePool -DFMDatabaseQueue=SF_FMDatabaseQueue -DFMResultSet=SF_FMResultSet -DFMDBBlockSQLiteCallBackFunction=SF_FMDBBlockSQLiteCallBackFunction $(inherited)
```

Comment:
Removed them in [this](https://github.com/bhariharan/SalesforceMobileSDK-iOS/commit/9263978ade2ffb6990635713e4f97e3dd28f6844) commit.